processor/awscontainerinsightprocessor/internal/stores/store.go

```diff
@@ -0,0 +1,25 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stores
+
+import "go.opentelemetry.io/collector/consumer/pdata"
+
+type K8sStore interface {
```

Comment:
Why is this interface defined in a package off by itself?  Shouldn't it be defined where it will be used?

Reply:
There will be more files in this folder. See the code here: https://github.com/aws-observability/aws-otel-collector/tree/container-insight-backup/internal/processor/awscontainerinsightprocessor/stores